docs/src/index.md

```diff
@@ -6,6 +6,33 @@ CurrentModule = TulipaEnergyModel
 
 [TulipaEnergyModel.jl](https://github.com/TulipaEnergy/TulipaEnergyModel.jl) is an optimization model for the electricity market that can be coupled with other energy sectors (e.g., hydrogen, heat, natural gas, etc.). The optimization model determines the optimal investment and operation decisions for different types of assets (e.g., producers, consumers, conversion, storage, and transport). _TulipaEnergyModel.jl_ is developed in [Julia](https://julialang.org/) and depends on the [JuMP.jl](https://github.com/jump-dev/JuMP.jl) package.
 
+## Tulipa in a Nutshell
```

Comment:
```suggestion
## TulipaEnergyModel in a Nutshell
```

Reply:
I think just Tulipa is more inviting. :)